src/lwc/gsChecklist/gsChecklist.html

```diff
@@ -0,0 +1,59 @@
+<template>
+    <article class="slds-setup-assistant__step">
+        <div class="slds-summary-detail">
+            <button class="slds-button slds-button_icon slds-m-right_x-small slds-m-top_x-small"
+                onclick={onClickOpenOrClose}
+                tabindex="-1"
+                title={group.title}>
+                <lightning-icon  
```

Comment:
In a screen reader context, would a user who cannot see these icons know the difference in what is being rendered on the screen and what would happen when they click the button? Basically in one case it's a ChevronRight icon and the other is ChevronDown, but I'm not seeing how that would be represented differently via a screen reader.

Reply:
I don't understand this comments, I never work to accessible and screen reader, I need a example to understand this

Reply:
@force2b

Reply:
I added alter text and aria-control, aria-expand, aria-hidden to button and icons

Reply:
@dcaballeroSf Who on the accessibility side is your team working with to validate behavior? If you'd like, I'll set up a meeting with your team and NGOhana and Crystal later this week to discuss development for accessibility. I also believe there were just a set of webinars on design, development and testing. I can find those for you as well if you'd like.

Reply:
There is a base `lightning-accordion` component that should solve all of this for you out of the box. Please refer to https://developer.salesforce.com/docs/component-library/bundle/lightning-accordion